client/my-sites/email/email-management/home/utils.js

```diff
@@ -167,3 +169,18 @@ export function resolveEmailPlanStatus( domain, emailAccount, isLoadingEmails )
 
 	return activeStatus;
 }
+
+/**
+ * Tracks an event for the key 'calypso_email_app_launch'. It will not track events from development mode.
+ *
+ * @param {app, context, provider} - app/context/provider should be string and must be provided.
+ */
+export function recordEmailActionTrackEvent( { app, context, provider } ) {
+	if ( ! isDevelopmentMode ) {
```

Comment:
I think we always want to track these events -- Tracks itself includes the UI and API environments in the events, so we can (and do) filter them out on the back end later. Not sending these events also makes it _really_ hard to debug the event logging!

Reply:
Done!